components/InPageNav.jsx

```diff
@@ -226,7 +226,7 @@ const InPageNav = (props) => {
     }, 100);
 
     const highlightCurrentSection = throttle(() => {
-        const windowTop = window.pageYOffset || document.documentElement.scrollTop;
+        const windowTop = window.scrollY || document.documentElement.scrollTop;
```

Comment:
my ide was telling me that pageyoffset was deprecated, i'll undo it

however w3schools and mdn sort of contradict a bit
https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollY
https://www.w3schools.com/jsref/prop_win_scrolly.asp#:~:text=The%20scrollY%20property%20is%20equal,scrollY.

if pageYoffset === scrollY then how is it any better for cross browser compatibility? (asking that of the articles, not of you) it's just confusing

Reply:
found this too - https://stackoverflow.com/a/46962472

Reply:
Several sources including w3schools state that pageYoffset is bettern for cross browser compatibility -

"The scrollY property is equal to the pageYOffset property.
For cross-browser compatibility, use window.pageYOffset instead of window.scrollY."

pageYOffset is more widely supported across older browsers.  

MDN says it's not deprecated but JetBrains IDEs and Microsoft documentation says it deprecated.   Let's follow MDN for now and keep pageYOffset